tensorflow/matmul.py

```diff
@@ -106,6 +114,8 @@ def main():
 
     result = {
         "@parameters": {
+            "framework": _ARGS_ML_FRAMEWORK,
+            "name": _ARGS_PI_NAME,
```

Comment:
As stated above, these do not look like parameters. And they can go one layer up in the dictionary in that case.